### DISCUSSION TITLE
Installer script for Nix-on-droid

### DISCUSSION THREAD

**Discussion by rairay91** [phoenixbyrd/Termux_XFCE#61] [Ideas]
Nix-on-droid is a termux based terminal which use nix package manger, and works out of the box.
But there is not a single tutorial about how to install and set up any de on nix-on-droid.
Why nix-on-droid - because it uses nix package manager, that mean we have access to every single package available for nixos. Which means every gaming tool like steam, lutris e.t.c. including proton layers.

**Comment by phoenixbyrd** [comment]
This is arm64? Idk anything about nix os
